inc/agent.class.php

```diff
@@ -296,6 +314,15 @@ public function handleAgent($metadata) {
          'itemtype'     => $metadata['itemtype'] ?? 'Computer'
       ];
 
+      $input['ip_version'] = 4;
+      if (isset($_SERVER['REMOTE_ADDR']) && filter_var($_SERVER['REMOTE_ADDR'], FILTER_VALIDATE_IP, FILTER_FLAG_IPV6)) {
+         $input['ip_version'] = 6;
+      }
```

Comment:
I don't see any usage of `ip_version` in your PR but there to just set it in database. I think you can just get rid of it.